server.go

```diff
@@ -2219,6 +2219,11 @@ func writeResponse(ctx *RequestCtx, w *bufio.Writer) error {
 		panic("BUG: cannot write timed out response")
 	}
 	err := ctx.Response.Write(w)
+	if err != nil {
+		if panicErr, ok := err.(*ErrBodyStreamWritePanic); ok {
+			ctx.logger.Printf("panic while writing response: %s", panicErr.Error())
```

Comment:
I don't think its necessary to log this here. The error will go all the way up to [workerpool.go#L230](https://github.com/valyala/fasthttp/blob/87c4546c7b1268a1c29c46e8f54def6f0e7596eb/workerpool.go#L230) where it will be logged anyways.